HIP/hip-820.md

```diff
@@ -71,38 +71,34 @@ When a dApp requires only the signature from the controller (wallet), it can use
 
 #### Returns
-`precheckCode` – an integer representing the [`ResponseCodeEnum`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/response_code.proto#L32) value returned from the Hedera Node, which may indicate success or failure.  A response code indicating success does not necessarily indicate the transaction will reach consensus nor succeed, it solely indicates whether the submitted transaction passed initial validation to warrant further processing.
-
 `transactionId` – a string encoded transaction identifier of the signed message that was sent to the Hedera node, in `<shard>.<realm>.<number>@<seconds>.<nanos>` format.
 
 `nodeId` – a string encoded transaction identifier of the Hedera Gossip Node's Account that the transaction was submitted to, in `<shard>.<realm>.<number>` format.
 
 `transactionHash`– a base64 encoding of the SHA384 digest of the [signedTransactionBytes](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction.proto#L68C19-L68C19) of the [Transaction](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction.proto#L43C13-L43C13) message that was submitted to the Hedera Network.
 
-### hedera_sendTransactionOnly
-When a dApp only requires the services of the controller to act as a relay to the Hedera network for submitting an already signed transaction, it can use the `hedera_sendTransactionOnly` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, transmits the message unaltered to the Hedera network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
+### hedera_executeTransaction
+When a dApp only requires the services of the controller to act as a relay to the Hedera network for submitting an already signed transaction, it can use the `hedera_executeTransaction` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, transmits the message unaltered to the Hedera network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
```

Comment:
Not interested in changing property name for same reasons stated above.

Reply:
@bugbytesinc understood. Let's see what other opinions there are and we can decide as a group how to move forward. Again, the idea here is to be more in line with the SDK's naming structure, which for new developers and those already utilizing the sdks, might relieve some of the burden of cognitive load.

here's a reference: https://github.com/hashgraph/hedera-improvement-proposal/pull/839#discussion_r1414368506